Example/Tests/AccelerateSolvePerfTest.swift

```diff
@@ -104,4 +105,34 @@ class AccelerateSolvePerfTest: XCTestCase {
             }
         }
     }
+
+    func transform(point: CGPoint, with transform: CATransform3D) -> CGPoint {
+        let vec = simd_double4(point) * Matrix4x4(transform)
+        let w = 1.0 // vec.w != 0 ? vec.w : Double.infinity
+        return CGPoint(x: vec.x / w, y: vec.y / w)
+    }
+
+    func testSimpleProjection() {
+        let src = Quadrilateral(CGRect(x: 1, y: 2, width: 3, height: 4))
+        let dst = Quadrilateral(CGRect(x: 1, y: 2, width: 13, height: 4))
+        let matrix = Perspective(src)
+            .projectiveTransform(destination: Perspective(dst))
+
+        let transformed = src.corners.map { transform(point: $0, with: matrix) }
```

Comment:
Instead of trying to compare CGRect after the transform would you like to compare CATransform3D 2 matrixes:
1. created by perspective transform
2. created by well known transformations like `CATransform3DMakeScale`, `CATransform3DTranslate`, etc